tests/end-to-end/director/application_api_test.go

```diff
@@ -984,6 +984,82 @@ func TestTenantSeparation(t *testing.T) {
 
 }
 
+func TestApplicationsForRuntime(t *testing.T) {
+	//Create first app in first tenant
+	//Given
```

Comment:
I think all the requests that you make to prepare the test should be in a single `// GIVEN` so there's only one of each: GIVEN/WHEN/THEN in your test. Also, maybe you could create those applications in a loop with a parametrized name and tenant? similar to `Applications` query test?

Reply:
I am going to create paramerized loop : )